media/js/base/global-init.js

```diff
@@ -11,5 +12,8 @@ $(document).ready(function() {
     initLangSwitcher();
     $(window).on('load', function () {
         $('html').addClass('loaded');
+        if (Mozilla.Client.isFirefoxDesktop) {
```

Comment:
Nit: please add a line of blank space above the conditional. A brief comment to give context as to what this is doing would be nice also.